mod/admin.php

```diff
@@ -909,6 +909,15 @@ function admin_page_summary(App $a)
 	$queues = ['label' => L10n::t('Message queues'), 'queue' => $queue, 'workerq' => $workerqueue];
 
 
+	$r = q("SHOW variables LIKE 'max_allowed_packet'");
```

Comment:
I didn't know how else to do this query. DBA::selectFirst would do a SELECT

Reply:
You did good, the DBA::select* methods have a very specific usage and it doesn't cover server variables.